test/framework/test_subscriber.hpp

```diff
@@ -272,6 +272,57 @@ namespace framework {
       bool on_complete_called = false;
     };
 
+    /**
+     * EqualToList checks subscribed values equals to vector<T>
+     * @tparam T - observable parameter
+     */
+    template <typename T>
+    class EqualToList : public VerificationStrategy<T> {
+    public:
+      /**
+       * @param expected_values - expected vector<T> values
+       * equals to list from observable
+       */
+      EqualToList(std::vector<T> expected_values)
+        : expected_values_(std::move(expected_values)) {}
```

Comment:
Do you mind to add move constructor as well an call it here?

Reply:
I didn't change to `EqualToList(std::vector<T> &&expected_values)` after all because, it reduces usability in test codes, and also they don't need performance.

Reply:
I mean to create additional constructor. Anyway it doesn't that really required

Reply:
> error: call of overloaded 'EqualToList(std::vector<int>)' is ambiguous
;(